Manifest

cheapsteak opened the pull request:
![image](https://cloud.githubusercontent.com/assets/743976/17562359/8f96403c-5ef8-11e6-9125-c9dd54eadc31.png)
- Added numberTween directive
  Usage: 

``` html
<number-tween
  value="number"
  formatter="optionalFormatterFunction"
  on-tween-start="optionalCallback"
  on-tween-end="optionalCallback"
></number-tween>
```
- Added lodash filter 
  Usage:

```
{{ input | _: 'methodName':'optionalAdditionalArgs' }}
```

e.g.

```
{{ array | _:'map':'property' | _:'sum' }}
```

btiernay commented:
Did you mean to have 2 "Software" pies?

cheapsteak commented:
![image](https://cloud.githubusercontent.com/assets/743976/16055564/e1356b24-323f-11e6-8aa6-9580c8cd84cd.png)

ok fixed again, replaced space with `&nbsp;`

cheapsteak commented:
charts hidden
![image](https://cloud.githubusercontent.com/assets/743976/16057096/6166fbfe-3246-11e6-8ac4-fdd2a84afc2f.png)

charts visible
![image](https://cloud.githubusercontent.com/assets/743976/16057113/75928ecc-3246-11e6-9242-d8d0d72d2b26.png)

btiernay commented:
Need to update the "units" for facets that relate to donors

slovit commented:
test this please

btiernay commented:
test this please

cheapsteak commented:
![image](https://cloud.githubusercontent.com/assets/743976/16211068/51369214-370d-11e6-8dfa-763883e30ecd.png)

Added bar chart for repositories

btiernay commented:
Looks good!

But:

```app/scripts/highcharts/js/directives.js
  line 722  col 141  Line is too long.

app/scripts/repository/js/repository.js
  line 543  col 32   'e' is defined but never used.```

btiernay commented:
A couple of suggestions:

- Drop the font size just enough so that `PCAWG Chicago` can be differentiated
- Move text slightly closer to axis and elide less characters (but preserve a bit of bottom padding)
- Maybe use grey instead of black for the labels?
- Maybe add one more major tick on y-axis

Keep in mind we may have 1 or 2 more repos to add soon.

cheapsteak commented:
Ready for further review & feedback

![image](https://cloud.githubusercontent.com/assets/743976/16247430/adbfb98c-37d7-11e6-98ed-83e8e1b4f779.png)

btiernay commented:
```
app/scripts/repository/js/repository.js
  line 901  col 11  Bad line breaking before '?'.
  line 915  col 6   Unnecessary semicolon.
```

cheapsteak commented:
Removed warnings, implemented clicking to toggle

cheapsteak commented:
Add cloud icons
![image](https://cloud.githubusercontent.com/assets/743976/16276564/e80f8c70-387c-11e6-8e32-89b5d72ce5f9.png)

btiernay commented:
@cheapsteak  Conflicts!!!!!

cheapsteak committed:
Add <number-filter/>

Usage:

<number-tween value="summary[repoData.repoName].donorCount || 0" filter="$filter('number')"></number-tween>

Make sure the scope has access to the $filter

cheapsteak committed:
numberTween: animate from old value instead of 0

cheapsteak committed:
Add onTweenStart and onTweenEnd to numberTween

cheapsteak committed:
Add lodash filter

usage:

{{ input | _: ‘methodName’:’optionalAdditionalArgs’ }}

cheapsteak committed:
Remove manifests column

cheapsteak committed:
Remove dots next to priority numbers. Align left (close to :::)

cheapsteak committed:
Only show numerator

cheapsteak committed:
Show manifest id button next to download button when only cloud repo selected

cheapsteak committed:
Add ‘totals' row

cheapsteak committed:
change namespace for numberTween to avoid collision

cheapsteak committed:
use numbertween

cheapsteak committed:
Remove donor totals. Rename “filter” to “formatter”

cheapsteak committed:
Add padding for right-most column

cheapsteak committed:
style: td transition all instead of only bgc

btiernay commented:
Can you please merge in develop so we get the count fix? Thanks!

cheapsteak committed:
Merge branch 'develop' of github.com:icgc-dcc/dcc-portal into feature/manifest-3
